test_django/pseudo_test/views.py

```diff
@@ -0,0 +1,36 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+from .models import Task
+import boto3   
+from test_django.settings import S3_NAME
+import random
+s3 = boto3.resource('s3')
+
+def send_answer(request):
+    if request.method == "POST":
+        uploaded_file = request.FILES['file']
+        if (uploaded_file.name[-4:] == ".txt" or uploaded_file.name[-4:] == ".pdc"
+            ) and uploaded_file.size < 1000000:
+            s3.Bucket(S3_NAME).put_object(
+            Key='{0}@{1}'.format(request.POST["task"],file_id()), Body=uploaded_file)
```

Comment:
Nie zwracasz file_id do użytkownika

Reply:
Do sesji ? Bo nie wiem za bardzo jak chcesz
 to wykorzystać (nie umiem JS)

Reply:
każdy plik który wrzucasz na s3 ma unikalny file_id, to jest identyfikator zlecenia. Lambda uruchami testy i wsadzi do bazy danych wyniki pod tym identyfikatorem. Jeśli użytkownik chce zobaczyć wyniki tych testów to musi podać ten identyfikator. Więc musi go skąd znać, a to ty go generujesz.